composer.json

```diff
@@ -116,7 +116,8 @@
     "allow-plugins": {
       "symfony/flex": true,
       "symfony/runtime": true,
-      "dealerdirect/phpcodesniffer-composer-installer": true
+      "dealerdirect/phpcodesniffer-composer-installer": true,
+      "endroid/installer": false
```

Comment:
Is there a specific reason for adding edroid installer? If there is one. Please specify this in the (to be written) PR description.

Reply:
This is a dependency of monitor-bundle introduced by Tacman. This is set when installing monitor-bundle in with composer. Not a introduction by this PR

Reply:
Ahh yes I kind of forgot about that. Then It's worrying that this change is part of your PR :man_shrugging:

Reply:
I answered with NO when the composer asked me to allow the recipe